bgpd/rfapi/rfapi_vty.c

```diff
@@ -850,12 +845,9 @@ int rfapiShowVncQueries(void *stream, struct prefix *pfx_match)
 		struct agg_node *rn;
 		int printedquerier = 0;
 
-
-		++nves_total;
-
 		if (rfd->mon
 		    || (rfd->mon_eth && skiplist_count(rfd->mon_eth))) {
```

Comment:
Can we simply the if condition here: since the if condition is no more required.
```
if (!rfd->mon &&
    !(rfd->mon_eth && skiplist_count(rfd->mon_eth)))
        continue;
```

Reply:
fixed